config-templates/ibm/ess/2.1.0/custom-parameters.json

```diff
@@ -0,0 +1,172 @@
+[
+	{
+		"description": "The mount path of the primary file system.",
+		"displayname": "Scale Host Path",
+		"name": "scale-host-path",
+		"required": "true",
+		"type": ""
+	},
+	{
+		"description": "The cluster ID of the pirmary IBM Spectrum Scale cluster.",
+		"displayname": "Cluster ID",
+		"name": "cluster-id",
+		"required": "true",
+		"regex": "^[0-9]+$",
+		"type": ""
+	},
+	{
+		"description": "The primary file system name.",
+		"displayname": "Primary FS",
+		"name": "primary-fs",
+		"required": "true",
+		"type": ""
+	},
+	{
+		"description": "The FQDN or IP address of the GUI node on the Spectrum Scale cluster.",
+		"displayname": "GUI Host",
+		"name": "gui-host",
+		"required": "true",
+		"type": ""
+	},
+	{
+		"description": "The name of secret for that contains the username and password to connect to the primary cluster GUI.",
+		"displayname": "Primary Cluster, Secret Name",
+		"name": "secret-name",
+		"required": "true",
+		"type": ""
+	},
+	{
+		"description": "The username to connect to the primary cluster GUI.",
+		"displayname": "Primary Cluster GUI Username",
+		"name": "gui-api-user",
+		"required": "true",
+		"category": "secret"
+	},
+	{
+		"description": "The password to connect to the primary cluster GUI.",
+		"displayname": "Primary Cluster GUI Password",
+		"name": "gui-api-password",
+		"required": "true",
+		"category": "secret"
```

Comment:
By default the secret is not obfuscated while displaying from CLI or GUI
It is recommended to obfuscate password by setting `"obfuscate": "true"`

Reply:
updated password to use obfuscate